2023/Tunde Babatunde/README.md

```diff
@@ -102,7 +101,7 @@ With this checklist, we expect you to demonstrate an ability to do research and
 
 - [x] Write a short blog post which demonstrates your understanding of **accessibility**. In English, with at least one image, less than 500 words. Demonstrate your understanding _based on past experiences_. For example, any specific accessibility improvements you’ve done in the past, or specific issues you identified. The more specific the better. The post has to be posted on a publicly-available platform (Dev.to, Hashnode, Medium, your own website), and you must also add it as a new section in your personal file in this repository (so we can provide feedback on the contents).
 
-> [Accessibility](https://medium.com/@motunrayocw/what-happens-when-you-type-something-on-google-search-bar-and-hit-enter-on-the-keyboard-73c42f0f4a10)
+> [Accessibility](https://medium.com/@motunrayocw/accessibility-329158b33db5)
```

Comment:
This is very generic information so doesn’t do much to demonstrate your expertise. Also make sure to add alt text for your images.